firefox-ios/Client/Frontend/Library/Bookmarks/Utitlity/BookmarksSaver.swift

```diff
@@ -8,7 +8,7 @@ import Shared
 
 protocol BookmarksSaver {
     /// Saves or updates a bookmark
```

Comment:
i'd Enhance this a bit, just to add that we are returning a GUID? that is nil for updates. Maybe we can set the signature of the method to return directly GUID? instead of String? so it is more clear, what do you think ?